### DISCUSSION TITLE
Fuel burn and climb rate

### DISCUSSION THREAD

**Discussion by CptBayer** [flybywiresim/aircraft#5456] [General]
Hi dev team,

at the moment fuel burn is much to high and climb power is very poor ... Is it an issue due to the current development state? No, it isnt.... take a look at the temperature ..... oMG .... 

![grafik](https://user-images.githubusercontent.com/76531893/127477019-f0697d61-227a-4e36-a057-50a68c934bd2.png)

greets klaus

**Comment by ExampleWasTaken** [comment]
> 
> 
> Hi dev team,
> 
> at the moment fuel burn is much to high and climb power is very poor ... Is it an issue due to the current development state? No, it isnt.... take a look at the temperature ..... oMG ....
> 
> ![grafik](https://user-images.githubusercontent.com/76531893/127477019-f0697d61-227a-4e36-a057-50a68c934bd2.png)
> 
> greets klaus

Live weather is bugged at the moment resulting in these kind of temperatures.
